packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -165,7 +164,7 @@ ReactDOMHydrationRoot.prototype.unmount = ReactDOMRoot.prototype.unmount = funct
 };
 
 export function createRoot(
-  container: Container,
```

Comment:
I assume that none of these prior uses of Container referred to `_reactRootContainer`

Reply:
I changed a call to `: any` for that reason. `_reactRootContainer` is only used by the legacy root renderer. It'll be removed when we remove legacy root.

It's also incomplete because the event system adds some stuff too which we'll remove once we get rid of the synthetic event system.